### DISCUSSION TITLE
Access to recipes from outside of the group

### DISCUSSION THREAD

**Discussion by tmllull** [mealie-recipes/mealie#1681] [General]
Hi! I've a question about a specific functionality for groups: "Allow users outside of your group to see your recipes".

I've this option activated in my main group (Home), and I'm trying to add some users in other groups, but no user (in other group) can see the recipes in the main group.

Is there any other option or step to take for this? I can't see anything in the docs or in the discussions.

Thanks a lot!

**Comment by hay-kot** [comment]
We're working on implementing a page for public recipes

- https://github.com/hay-kot/mealie/pull/1641

In the nightly version there is an option to copy a public link from recipes.

Here's the issue we're using to track this feature

- https://github.com/hay-kot/mealie/issues/1326
